SpatialGDK/Source/SpatialGDK/Private/Interop/GlobalStateManager.cpp

```diff
@@ -595,10 +595,36 @@ void UGlobalStateManager::BecomeAuthoritativeOverAllActors()
 	}
 }
 
+void UGlobalStateManager::BecomeAuthoritativeOverActorsBasedOnLBStrategy()
+{
+	for (TActorIterator<AActor> It(NetDriver->World); It; ++It)
+	{
+		AActor* Actor = *It;
+		if (Actor != nullptr && !Actor->IsPendingKill())
```

Comment:
Incredibly minor and I'm sorry if you know about it, but I've heard that weak UObject pointers internally return null for actors that are pending kill / GC. Is it neater to locally instance the iterated var to one of those and then have the check implicitly done on dereferencing it?